organizations/tests/test_api.py

```diff
@@ -522,20 +541,51 @@ def test_edge_cases(self):
             organization["name"] for organization in organizations
         }
 
+        # Based on logging messages, make sure we dropped the appropriate
+        # organization dict from the bulk-add batch.
+        logging_of_drop_from_batch = mock_log_info.call_args_list[0][0]
+        assert logging_of_drop_from_batch[1]["short_name"] == (
+            # We dropped this org data:
+            self.make_organization_data("ORG_x")["short_name"]
+        )
+        assert logging_of_drop_from_batch[2]["short_name"] == (
+            # in favor of this org data, which came earlier in the batch.
+            self.make_organization_data("org_X")["short_name"]
+        )
+
+        # Based on logging messages, make sure the expected breakdown of
+        #    created vs. reactivated vs. not touched
+        # is true for the organizations passed to `bulk_add_organizations`.
+        logged_orgs_to_reactivate = mock_log_info.call_args_list[1][0][2]
+        assert set(logged_orgs_to_reactivate) == {
+            "org_to_reactivate"
+        }
+        logged_orgs_to_create = mock_log_info.call_args_list[2][0][2]
+        assert set(logged_orgs_to_create) == {
+            "org_X", "org_Y"
+        }
+
```

Comment:
So normally I'm against testing logging messages this specifically; IMO it generally makes tests more brittle with little benefit.

However, the correctness of these log messages is quite important, as `./manage.py cms backfill_orgs_and_org_courses --dry-run` will depend on them in order to show the user what's about to be bulk-created/bulk-reactivated.